modules/SofaGeneralAnimationLoop/src/SofaGeneralAnimationLoop/MechanicalMatrixMapper.h

```diff
@@ -32,49 +32,15 @@
 
 #include <sofa/core/topology/BaseMeshTopology.h>
 
-#include <sofa/simulation/MechanicalVisitor.h>
-
-#include <sofa/core/MultiVecId.h>
-#include <sofa/core/BaseMapping.h>
 #include <sofa/linearalgebra/SparseMatrixProduct[EigenSparseMatrix].h>
 
-namespace sofa::component::interactionforcefield
+namespace sofa::simulation
```

Comment:
It is better to not forward declare things manually, including sofa/simulation/fwd.h.